project/Dependencies.scala

```diff
@@ -1,45 +1,35 @@
 import sbt._
-
+import Versions._
+import ExclusionRules._
 object Dependencies {
-  val excludeCglib = ExclusionRule(organization = "org.sonatype.sisu.inject")
-  val excludeJackson = ExclusionRule(organization = "org.codehaus.jackson")
-  val excludeScalaTest = ExclusionRule(organization = "org.scalatest")
-  val excludeScala= ExclusionRule(organization = "org.scala-lang")
-  val excludeNettyIo = ExclusionRule(organization = "io.netty", artifact= "netty-all")
-  val excludeAsm = ExclusionRule(organization = "asm")
-  val excludeQQ = ExclusionRule(organization = "org.scalamacros")
-  val jdkVersion = scala.util.Properties.isJavaAtLeast("1.8")
 
-  lazy val typeSafeConfigDeps = if(jdkVersion){
-    "com.typesafe" % "config" % "1.3.0"
-  } else {
-    "com.typesafe" % "config" % "1.2.1"
-  }
-  lazy val yammerDeps = "com.yammer.metrics" % "metrics-core" % "2.2.0"
+  lazy val typeSafeConfigDeps = "com.typesafe" % "config" % typeSafeConfig
+
+  lazy val yammerDeps = "com.yammer.metrics" % "metrics-core" % metrics
 
   lazy val jodaDeps = Seq(
-    "org.joda" % "joda-convert" % "1.8.1",
-    "joda-time" % "joda-time" % "2.9.3"
+    "org.joda" % "joda-convert" % jodaConvert,
+    "joda-time" % "joda-time" % jodaTime
   )
 
   lazy val akkaDeps = Seq(
     // Akka is provided because Spark already includes it, and Spark's version is shaded so it's not safe
     // to use this one
-    "com.typesafe.akka" %% "akka-slf4j" % "2.3.15" % "provided",
-    "com.typesafe.akka" %% "akka-cluster" % "2.3.15" exclude("com.typesafe.akka", "akka-remote"),
-    "io.spray" %% "spray-json" % "1.3.2",
-    "io.spray" %% "spray-can" % "1.3.3",
-    "io.spray" %% "spray-caching" % "1.3.3",
-    "io.spray" %% "spray-routing" % "1.3.3",
-    "io.spray" %% "spray-client" % "1.3.3",
+    "com.typesafe.akka" %% "akka-slf4j" % akka % "provided",
+    "com.typesafe.akka" %% "akka-cluster" % akka exclude("com.typesafe.akka", "akka-remote"),
+    "io.spray" %% "spray-json" % sprayJson,
```

Comment:
Not a biggie, but I wonder if `Version.sprayJson` etc, would be more readable.  Probably not.